e2e/wdio/core/tests/object-identifier.e2e-spec.ts

```diff
@@ -0,0 +1,46 @@
+import { ObjectIdentifierPo } from '../pages/object-identifier.po';
+import {
+    getElementArrayLength,
+    isElementClickable,
+    refreshPage,
+    scrollIntoView,
+    waitForPresent
+} from '../../driver/wdio';
+
+describe('Object identifier test suite', function() {
+    const objectIdentifierPage = new ObjectIdentifierPo();
+    const {
+        identifier,
+        clickableLinks,
+    } = objectIdentifierPage;
+
+    beforeAll(() => {
+        objectIdentifierPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForPresent(identifier);
+    }, 1);
+
+    it('Verify each identifier is clickable', () => {
+        const linkElementArr = getElementArrayLength(clickableLinks);
+        for (let i = 0; i < linkElementArr; i++) {
+            scrollIntoView(clickableLinks, i);
+            expect(isElementClickable(clickableLinks, i)).toBe(true);
```

Comment:
please add failure message: 'link with index ${i} not clickable'

Reply:
Done

Reply:
@nikvalor please mark as resolved